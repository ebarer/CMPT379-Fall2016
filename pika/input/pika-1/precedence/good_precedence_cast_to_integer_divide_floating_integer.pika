exec {

  print [ 8.4 | int ] / 38, _n_.
  print [ 2.9 | int ] / 75, _n_.
  print [ 1.3 | int ] / 49, _n_.
  print [ 9.0 | int ] / 31, _n_.
  print [ 9.3 | int ] / 5, _n_.
  print [ 2.1 | int ] / 28, _n_.
  print [ 1.3 | int ] / 69, _n_.
  print [ 3.1 | int ] / 74, _n_.
  print [ 2.6 | int ] / 49, _n_.
  print [ 6.6 | int ] / 84, _n_.
  print [ 5.8 | int ] / 10, _n_.
  print [ 7.6 | int ] / 14, _n_.

  print 45 / [ 7.7 | int ], _n_.
  print 49 / [ 1.4 | int ], _n_.
  print 67 / [ 8.6 | int ], _n_.
  print 33 / [ 2.3 | int ], _n_.
  print 30 / [ 2.3 | int ], _n_.
  print 49 / [ 3.7 | int ], _n_.
  print 48 / [ 5.5 | int ], _n_.
  print 46 / [ 4.0 | int ], _n_.
  print 55 / [ 9.9 | int ], _n_.
  print 64 / [ 4.1 | int ], _n_.
  print 70 / [ 9.0 | int ], _n_.
  print 56 / [ 1.1 | int ], _n_.

  print [ 8.5 | int ] / [ 9.0 | int ], _n_.
  print [ 5.5 | int ] / [ 4.2 | int ], _n_.
  print [ 6.9 | int ] / [ 3.2 | int ], _n_.
  print [ 3.0 | int ] / [ 7.2 | int ], _n_.
  print [ 5.6 | int ] / [ 2.1 | int ], _n_.
  print [ 5.8 | int ] / [ 2.7 | int ], _n_.
  print [ 7.8 | int ] / [ 3.4 | int ], _n_.
  print [ 7.9 | int ] / [ 5.5 | int ], _n_.
  print [ 4.2 | int ] / [ 1.6 | int ], _n_.
  print [ 4.3 | int ] / [ 5.7 | int ], _n_.
  print [ 9.8 | int ] / [ 7.2 | int ], _n_.
  print [ 8.6 | int ] / [ 1.3 | int ], _n_.

}

