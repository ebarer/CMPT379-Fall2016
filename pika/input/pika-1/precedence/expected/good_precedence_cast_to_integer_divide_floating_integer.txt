0
0
0
0
1
0
0
0
0
0
0
0
6
49
8
16
15
16
9
11
6
16
7
56
0
1
2
0
2
2
2
1
4
0
1
8
